includes/views/listing_contact.php

```diff
@@ -55,6 +57,18 @@ private function validate() {
             $this->errors[] = _x( 'Please enter a valid email.', 'contact-message', 'business-directory-plugin' );
         }
 
+        /**
+         * Optional phone validation.
+         *
+         * @param bool $validate - defaults to false to skip phone validation.
+         * @param string $phone - the phone field.
+         *
+         * @return bool
+         */
+        if ( apply_filters( 'wpbdp_contact_form_validate_phone', false, $this->phone ) ) {
+            $this->errors[] = _x( 'Please enter a valid phone number.', 'contact-message', 'business-directory-plugin' );
```

Comment:
Oh, let's add format validation here when the field isn't blank. Here's the regex we're using in Formidable:
https://github.com/Strategy11/formidable-forms/blob/a6b06bcc7a0fe6a3ab5cdc0ea848f79697b960ad/classes/models/FrmEntryValidate.php#L177

Reply:
Sounds good, I know where to put it :)